packages/table/src/table.tsx

```diff
@@ -530,14 +558,31 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     private bodyCellRenderer = (rowIndex: number, columnIndex: number) => {
-        return this.getColumnProps(columnIndex).renderCell(rowIndex, columnIndex);
+        const columnProps = this.getColumnProps(columnIndex);
+        const cell = columnProps.renderCell(rowIndex, columnIndex);
+        const cellLoading = cell.props.loading;
+
+        let loading: boolean;
+        if (cellLoading != null) {
+            loading = cellLoading;
+        } else {
+            const hasLoadingOption = this.hasLoadingOption(columnProps.loadingOptions, ColumnLoadingOption.CELLS);
+            if (hasLoadingOption) {
+                loading = true;
+            } else if (hasLoadingOption === false) {
+                loading = false;
+            }
+        }
```

Comment:
and again with the direct assignment https://github.com/palantir/blueprint/pull/473/files#r98076641